assets/css/main.css

```diff
@@ -0,0 +1,214 @@
+@font-face {
+  font-family: 'Gotham';

```

Comment:
Could you show us renderings with this font? Our font as stated in our branding guide are either Montserrat or Track, but I want to see what this would look like.

Reply:
I'll send you screenshots on Slack.